bundles/org.openhab.binding.nibeheatpump/README.md

```diff
@@ -62,16 +62,15 @@ An Arduino-based solution has been tested with Arduino uno + RS485 and Ethernet
 [PRODINo ESP32 Ethernet v1](https://kmpelectronics.eu/products/prodino-esp32-ethernet-v1/) and [ProDiNo Ethernet V2](https://kmpelectronics.eu/products/prodino-ethernet-v2/) boards are also supported.
 PRODINo boards have built-in Ethernet and RS-485 ports.
 
-Arduino code is available [here](https://github.com/openhab/openhab-addons/tree/main/bundles/org.openhab.binding.nibeheatpump/contrib/NibeGW/Arduino/NibeGW).
+The Arduino code for NibeGW is available in the [NibeGW Arduino Directory on GitHub repository](https://github.com/openhab/openhab-addons/tree/main/bundles/org.openhab.binding.nibeheatpump/contrib/NibeGW/Arduino/NibeGW).
 
-Arduino code can be build via Arduino IDE.
-For more details see [www.arduino.cc](https://www.arduino.cc/en/Main/Software).
-NibeGW configuration (such IP addresses, ports, etc) can be adapted directly by editing the code files.
+Arduino code can be built via the [Arduino IDE software download page](https://www.arduino.cc/en/Main/Software).
```

Comment:
It seems more explicit in the previous version where it was mentioned that Arduino code can be built by using the IDE. Even it was was slightly awkwardly phrased, the code cannot by built via a download page.